modules/openapi-generator/src/main/java/org/openapitools/codegen/examples/ExampleGenerator.java

```diff
@@ -400,6 +391,22 @@ private void traverseSchemaProperties(String mediaType, Schema schema, Set<Strin
                 Schema property = (Schema) schema.getProperties().get(propertyName.toString());
                 values.put(propertyName.toString(), resolvePropertyToExample(propertyName.toString(), mediaType, property, processedModels));
             }
+        } else if (ModelUtils.isAllOf(schema) || ModelUtils.isAllOfWithProperties(schema)) {
+            resolveAllOfSchemaProperties(mediaType, schema, processedModels, values);
+        }
+    }
+
+    private void resolveAllOfSchemaProperties(String mediaType, Schema schema, Set<String> processedModels, Map<String, Object> values) {
```

Comment:
can you please add a 1-liner docstring explaining what this function does?

Reply:
Done